posts/2023-07-24_rounding/rounding.qmd

```diff
@@ -25,8 +25,7 @@ long_slug <- "2023-07-24_rounding"
 Both SAS and base R have the function `round()`, which rounds the input to the specified number of decimal places.
 However, they use different approaches when rounding off a 5:
 
--   SAS `round()` [rounds half up](https://en.wikipedia.org/wiki/Rounding#Rounding_half_up).
-    This is the most common method of rounding.
+-   SAS `round()` [rounds away from zero](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/lefunctionsref/p0tj6cmga7p8qln1ejh6ebevm0c9.htm#p0zr4tmk27unjwn1izg4801h5l46). So, `1.5` would be rounded to 2 whereas `-1.5` to `-2`.
```

Comment:
```suggestion
-   SAS `round()` [rounds away from zero](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/lefunctionsref/p0tj6cmga7p8qln1ejh6ebevm0c9.htm#p0zr4tmk27unjwn1izg4801h5l46). So, `0.5` would be rounded to 1 whereas `-1.5` to `-2`.
```
The 2 examples given here currently would give the same result in SAS and R so maybe better to use one that more highlights the difference

Reply:
this would also then be consistent with the examples in the below bullet